python/ray/train/_internal/storage.py

```diff
@@ -553,6 +553,13 @@ def persist_current_checkpoint(self, checkpoint: "Checkpoint") -> "Checkpoint":
                 dest_fs=self.storage_filesystem,
             )
         )
+
+        # Raise an error if the storage path is not accessible when
+        # attempting to upload a checkpoint from a remote worker.
+        # Ex: If storage_path is a local path, then a validation marker
+        # will only exist on the head node but not the worker nodes.
+        self._check_validation_file()
+
```

Comment:
Note: This is a change that's needed so that multi-node w/ no storage path doesn't fail when NO checkpoints are reported.

Reply:
@ericl This is the same raise error behavior that we added for head node syncing in 2.6.